packages/beacon-node/src/chain/blocks/types.ts

```diff
@@ -68,7 +68,8 @@ export type CachedBlobs = CachedDataItem &
  */
 export enum DataColumnsSource {
   gossip = "gossip",
-  api = "api",
+  beaconApi = "beacon_api",
+  engineApi = "engine_api",
```

Comment:
we could keep previous name and use the following naming here but not sure it's better

```suggestion
  api = "api",
  engine = "engine",
```